plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeAnalyzeProperties.java

```diff
@@ -62,7 +71,13 @@ public DeltaLakeAnalyzeProperties()
                         null,
                         false,
                         DeltaLakeAnalyzeProperties::decodeColumnNames,
-                        value -> value));
+                        value -> value),
+                enumProperty(
+                        MODE_PROPERTY,
+                        "Analyze mode, INCREMENTAL or FULL_REFRESH",
```

Comment:
This argument is a description prefix. `enumProperty` method appends possible values. 
```suggestion
                        "Analyze mode",
```